### DISCUSSION TITLE
Please change the url

### DISCUSSION THREAD

**Discussion by kit1211** [langflow-ai/langflow#3875] [Ideas]
Please change the url

![image](https://github.com/user-attachments/assets/f35eb415-2244-458e-a956-fe0958a0edaa)
![image](https://github.com/user-attachments/assets/150c84f9-8c5f-4e3b-baea-424ad7cd29e5)

**Comment by severfire** [comment]
i think it could go to issues / bugs

**Comment by carlosrcoelho** [comment]
@kit1211 

Thank you!

It'll be merged soon.

https://github.com/langflow-ai/langflow/pull/3966
